<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trace_form</class>
 <widget class="QDialog" name="trace_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>913</x>
     <y>551</y>
     <width>78</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>30</y>
     <width>78</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QTableView" name="wh_view">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>382</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="condition_view">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>382</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="spac_view">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>382</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="trace_view">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>112</y>
     <width>961</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>35</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>362</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Warehouse Changes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>362</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Condition Changes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>362</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Spec Changes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>92</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Purchased/Sold/Returned on:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>65</y>
     <width>65</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="description_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>65</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>trace_form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
